test/Microsoft.AspNetCore.Mvc.FunctionalTests/FormatFilterSampleTest.cs

```diff
@@ -42,8 +43,15 @@ public FormatFilterSampleTest(MvcTestFixture<FormatFilterSample.Web.Startup> fix
         [Fact]
         public async Task FormatFilter_ExtensionInRequest_Optional()
         {
-            // Arrange & Act
-            var response = await Client.GetAsync("http://localhost/FormatFilter/GetProduct.json");
+            // Arrange
+            var request = new HttpRequestMessage();
+            // Typical accept header sent by Chrome browser
+            request.Headers.Add("Accept", "text/html,application/xhtml+xml," +
+                "application/xml;q=0.9,image/webp,*/*;q=0.8");
```

Comment:
Please move the comment and the `string` to a `const`. Just use the `const` here and in the other tests.